gke-to-gke-peering/cleanup.sh

```diff
@@ -77,23 +70,23 @@ kubectl delete -f "$ROOT"/manifests/run-my-nginx.yaml
 sleep 120
 
 ### Delete clusters
-gcloud deployment-manager deployments delete cluster1-deployment --project "$1" \
+gcloud deployment-manager deployments delete cluster1-deployment \
```

Comment:
While we're in here, we can group all these deployments into a single command: `gcloud deployment-manager deployments delete cluster1-deployment cluster2-deployment cluster3-deployment cluster4-deployment` to speed up the script.